diff -ruN src.new/HTTPDownload.cpp src/HTTPDownload.cpp
--- src.new/HTTPDownload.cpp	2022-05-02 21:18:27.956149797 +0800
+++ src/HTTPDownload.cpp	2022-05-02 21:25:08.986147464 +0800
@@ -28,14 +28,6 @@
 #	include "config.h"				// Needed for HAVE_LIBCURL
 #endif
 
-#ifdef HAVE_LIBCURL
-#	include <curl/curl.h>
-#	include <common/ClientVersion.h>		// Needed for the VERSION_ defines
-#	include <common/MacrosProgramSpecific.h>	// Needed for GUI_ONLY()
-#else
-#	include <wx/protocol/http.h>
-#endif
-
 #include <wx/wfstream.h>				// Needed for wxFFileOutputStream
 #include "HTTPDownload.h"				// Interface declarations
 #include <common/StringFunctions.h>		// Needed for unicode2char
@@ -47,6 +39,13 @@
 #include "ScopedPtr.h"
 #include <wx/filename.h>				// Needed for wxFileName
 
+#ifdef HAVE_LIBCURL
+#	include <curl/curl.h>
+#	include <common/ClientVersion.h>		// Needed for the VERSION_ defines
+#	include <common/MacrosProgramSpecific.h>	// Needed for GUI_ONLY()
+#else
+#	include <wx/protocol/http.h>
+#endif
 
 #ifndef AMULE_DAEMON
 #include "inetdownload.h"	// Needed for inetDownload
diff -ruN src.new/LibSocketAsio.cpp src/LibSocketAsio.cpp
--- src.new/LibSocketAsio.cpp	2022-05-02 21:18:27.886149797 +0800
+++ src/LibSocketAsio.cpp	2022-05-02 21:18:57.996149622 +0800
@@ -25,6 +25,8 @@
 
 #include "config.h"		// Needed for HAVE_BOOST_SOURCES
 
+#include <wx/thread.h>		// wxMutex
+#include <wx/intl.h>		// _()
 
 #ifdef _MSC_VER
 #define _WIN32_WINNT 0x0501	// Boost complains otherwise
@@ -57,8 +59,6 @@
 #endif
 
 #include "LibSocket.h"
-#include <wx/thread.h>		// wxMutex
-#include <wx/intl.h>		// _()
 #include <common/Format.h>	// Needed for CFormat
 #include "Logger.h"
 #include "GuiEvents.h"
diff -ruN src.new/UPnPBase.cpp src/UPnPBase.cpp
--- src.new/UPnPBase.cpp	2022-05-02 21:18:27.886149797 +0800
+++ src/UPnPBase.cpp	2022-05-02 21:18:57.996149622 +0800
@@ -27,13 +27,13 @@
 
 #ifdef ENABLE_UPNP
 
+#include "UPnPBase.h"
 // check for broken Debian-hacked libUPnP
 #include <upnp.h>
 #ifdef STRING_H				// defined in UpnpString.h Yes, I would have liked UPNPSTRING_H much better.
 #define BROKEN_DEBIAN_LIBUPNP
 #endif
 
-#include "UPnPBase.h"
 
 #include <algorithm>		// For transform()
 
diff -ruN src.new/UPnPBase.h src/UPnPBase.h
--- src.new/UPnPBase.h	2022-05-02 21:18:27.916149797 +0800
+++ src/UPnPBase.h	2022-05-02 21:18:57.996149622 +0800
@@ -29,11 +29,11 @@
 #define AMULE_UPNP_H
 
 
+#include "UPnPCompatibility.h"
 #include <map>
 #include <string>
 #include <sstream>
 
-#include "UPnPCompatibility.h"
 
 #include <common/SmartPtr.h>		// Needed for CSmartPtr
 
diff -ruN src.new/UPnPCompatibility.h src/UPnPCompatibility.h
--- src.new/UPnPCompatibility.h	2022-05-02 21:18:27.896149797 +0800
+++ src/UPnPCompatibility.h	2022-05-02 21:18:57.996149622 +0800
@@ -27,6 +27,8 @@
 #ifndef UPNPCOMPATIBILITY_H
 #define UPNPCOMPATIBILITY_H
 
+#include <common/MuleDebug.h>		// for CMuleException::
+#include <common/StringFunctions.h>	// for char2unicode()
 
 #include <upnp.h>
 #include <upnptools.h>
@@ -35,8 +37,6 @@
 #include <sstream>
 
 
-#include <common/MuleDebug.h>		// for CMuleException::
-#include <common/StringFunctions.h>	// for char2unicode()
 
 
 #include "Logger.h"				// for Add(Debug)LogLineM()
