--- configure	2022-01-10 00:02:47.576270047 +0900
+++ configure	2022-01-10 00:11:08.087933444 +0900
@@ -9293,7 +9293,6 @@
 fi
 
 
-	if test $cross_compiling = no; then :
 
 		{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for libupnp version >= 1.6.6" >&5
 $as_echo_n "checking for libupnp version >= 1.6.6... " >&6; }
@@ -9345,17 +9344,8 @@
 
 	if test ${result:-no} = yes; then :
   LIBUPNP_CPPFLAGS="$LIBUPNP_CPPFLAGS -DENABLE_UPNP=1"
-else
-  enable_upnp=disabled
-		{ $as_echo "$as_me:${as_lineno-$LINENO}: WARNING:
-        UPnP code has been disabled because ${libupnp_error}." >&5
-$as_echo "$as_me: WARNING:
-        UPnP code has been disabled because ${libupnp_error}." >&2;}
-_mule_warning_5=yes
-_mule_has_warnings=yes
 
 
-fi
 
 
 
