diff --git a/library/platform.c b/library/platform.c
index b15b7b2..bf87ba3 100644
--- a/library/platform.c
+++ b/library/platform.c
@@ -133,7 +133,7 @@ int mbedtls_platform_win32_vsnprintf(char *s, size_t n, const char *fmt, va_list
         return -1;
     }
 
-#if defined(_TRUNCATE)
+#if defined(_TRUNCATE) && !defined(__MINGW32__)
     ret = vsnprintf_s(s, n, _TRUNCATE, fmt, arg);
 #else
     ret = vsnprintf(s, n, fmt, arg);
